decidim-core/lib/decidim/form_builder.rb

```diff
@@ -181,7 +181,7 @@ def editor(name, options = {})
 
       content_tag(:div, class: "editor #{"hashtags__container" if options[:hashtaggable]}") do
         template = ""
-        template += label(name, options[:label].to_s || name) + required_for_attribute(name) if options[:label] != false
+        template += label(name, label_for(name) + required_for_attribute(name)) unless options[:label] == false
```

Comment:
Maybe using `if options.fetch(:label, true)` would be clearer?

Reply:
👍 Ok good suggestion, I'll change it